src/main/java/frc/team4909/robot/subsystems/drivetrain/commands/Drive.java

```diff
@@ -12,8 +12,8 @@ public Drive() {
     public void execute() {
         // Calls tank function using left Y and right Y
         // Negated since Y Axis is scaled -1 to +1 top to bottom (counterintuively)
-        Robot.drivetrainSubsystem.tankDrive(-Robot.driverGamepad.getThresholdAxis(BionicF310.LY),
-                -Robot.driverGamepad.getThresholdAxis(BionicF310.RY));
+        Robot.drivetrainSubsystem.arcadeDrive(-Robot.driverGamepad.getRawAxis(1),
+                -Robot.driverGamepad.getRawAxis(4));
```

Comment:
why raw axis not threshold..? the reason we use threshold is so that we ignore the "deadband" around the center (resting is like 0.062) or something similar... if you truly would like raw axes, I'd recommend creating a method on BionicJoystick so we can pass in BionicF310.LY to that as well, or at the very least comment what axes here, because the comments on line 13 is outdated.